### PR TITLE
Fix batch update button / checkboxes

### PR DESCRIPTION
**Pull request by Averus-a**
### Description of Change ###

- Checking single item (without using 'select all' checkbox) now forces update button availability
- 'select all' check is no more de-sync with checkboxes inside package list

### Issues Resolved ### 

- fixes #

### API Changes ###

 
None

### Platforms Affected ### 

- All

### Behavioral Changes ###

None

### PR Checklist ###

- [ ] I have included examples or tests
- [ ] I have updated the change log
- [ ] I am listed in the CONTRIBUTORS file
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

File: src/Orc.NuGetExplorer.Xaml/Constants.cs [hunk 1]
```diff
@@ -43,4 +43,13 @@ internal static class NuGetExplorer
             public const string LastRepositoryDefaultValue = RepositoryName.All;
         }
     }
+
+    internal static class Commands
+    {
+        internal static class Page
+        {
+            public const string BatchUpdatePackages = $"{nameof(Page)}.{nameof(BatchUpdatePackages)}";
```
Comment (GeertvanHorrik) [inline src/Orc.NuGetExplorer.Xaml/Constants.cs:51]: rename to Extensions.BatchUpdatePackages

File: src/Orc.NuGetExplorer.Xaml/Commands/PageBatchUpdatePackagesCommandContainer.cs [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    using System.Threading.Tasks;
+    using Catel;
+    using Catel.Logging;
+    using Catel.MVVM;
+    using Catel.Services;
+    using Orc.NuGetExplorer.Management;
+    using Orc.NuGetExplorer.Packaging;
+    using Orc.NuGetExplorer.Windows;
+
+    internal class PageBatchUpdatePackagesCommandContainer : CommandContainerBase<IManagerPage>
```
Comment (GeertvanHorrik) [inline src/Orc.NuGetExplorer.Xaml/Commands/PageBatchUpdatePackagesCommandContainer.cs:16]: Rename to ExtensionsBatchUpdatePackagesCommandContainer

File: src/Orc.NuGetExplorer.Xaml/Commands/PageBatchUpdatePackagesCommandContainer.cs [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+            try
+            {
+                _progressManager.ShowBar(parentVM);
+
+                var batchedPackages = sourcePage.PackageItems.Where(x => x.IsChecked).ToList();
+
+                if (batchedPackages.Any(x => x.ValidationContext.HasErrors))
+                {
+                    await _messageService.ShowErrorAsync("Can't perform update. One or multiple package cannot be updated due to validation errors", "Can't update packages");
```
Comment (GeertvanHorrik) [inline src/Orc.NuGetExplorer.Xaml/Commands/PageBatchUpdatePackagesCommandContainer.cs:71]: One or more package(s) cannot be updated due to validation errors

**Review by GeertvanHorrik — CHANGES_REQUESTED** [review]

File: src/Orc.NuGetExplorer.Xaml/Commands/PageBatchUpdatePackagesCommandContainer.cs (hunk 2 above)
Comment (mkhomutov) [reply src/Orc.NuGetExplorer.Xaml/Commands/PageBatchUpdatePackagesCommandContainer.cs:16]: We have Orc.Extensibility for the Extensions things. NuGetExplorer is now using not only in the projects with Extensions.
My opinion is using "Extensions" word in Orc.NuGetExplorer is not a good idea

**Review by mkhomutov — COMMENTED** [review]
Just a small comment to the comment :)
in general looks good for me

**Review by mkhomutov — COMMENTED** [review]
I like it, thanks

**Review by GeertvanHorrik — APPROVED** [review]

**Comment by Averus-a** [comment]
@GeertvanHorrik 
Pushed some tests, can you check, please?

File: src/Orc.NuGetExplorer.Tests/ViewModels/PageActionBarViewModelFacts.cs [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+    [TestFixture]
+    internal class PageActionBarViewModelFacts
+    {
+        public class TheCheckAllExecuteAsyncMethod
+        {
+            [TestCase]
+            public async Task InvalidatesPackagesBatchUpdateCommandAsync()
+            {
+                var serviceLocator = ServiceLocator.Default;
```
Comment (GeertvanHorrik) [inline src/Orc.NuGetExplorer.Tests/ViewModels/PageActionBarViewModelFacts.cs:24]: create a copy / clone for unit testing.

**Review by GeertvanHorrik — CHANGES_REQUESTED** [review]

**Pull request merged**